db/migrate/20170217085547_add_type_to_cloud_object_store_container.rb

```diff
@@ -0,0 +1,17 @@
+class AddTypeToCloudObjectStoreContainer < ActiveRecord::Migration[5.0]
+  class CloudObjectStoreContainer < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    add_column :cloud_object_store_containers, :type, :string
+    add_index :cloud_object_store_containers, [:type, :ems_id]
+
+    CloudObjectStoreContainer.reset_column_information
```

Comment:
This should not be necessary.  If it is, then there is some sort of contamination going on.  If you only see it in tests, then you may want to split creating the column and doing the data migration into two separate migrations.